File: Hackathon-plot2.R
library('rworldmap')

df<-read.csv("owid-covid-data.csv")
df<-filter(df, location!="World")
df<-filter(df, location!="International")
df<-filter(df, iso_code!="")
df<-filter(df, date=="2020-07-10")
mapped_data <- joinCountryData2Map(df, joinCode = "ISO3", 
                                   nameJoinColumn = "iso_code")
par(mai=c(0,0,0.2,0),xaxs="i",yaxs="i")
mapCountryData(mapped_data, nameColumnToPlot = "total_cases_per_million")

File: Hackathon-plot2.py
import pandas as pd
import plotly.express as px

df=pd.read_csv("owid-covid-data.csv")
df=df[df['location']!= "World" ]
df=df[df['location']!= "International"]
df=df[df['iso_code']!=" "]
df=df[df["date"]=="2020-07-10"]
fig=px.choropleth(df,locations="iso_code",color="total_cases_per_million",hover_name="location",
    color_continuous_scale=px.colors.sequential.deep)


fig.show()